hbase-shell/src/main/ruby/shell/commands/alter.rb

```diff
@@ -82,6 +82,12 @@ def help
 
   hbase> alter 't1', METHOD => 'table_att_unset', NAME => 'coprocessor$1'
 
+Other than removing coprocessor from the table-scope attribute via 'table_att_unset', you can also
```

Comment:
nit: Can we mention that multiple CP names can be passed as parameter?

Reply:
good suggestion, I added it and we became the first to show how we can remove multiple target as array in this command help page :P